### DISCUSSION TITLE
Add entries to "included_apps.txt" after deploying with MSI from Intune.

### DISCUSSION THREAD

**Discussion by Jcgull** [Romanitho/Winget-AutoUpdate#755] [General]
Hi! I am wondering about how i can add AppID's to my "included_apps.txt" file i deployed with my MSI file in Intune. Do i have to edit the file, repackage it, and reinstall it on the clients to add entries here? Or is it a simple way to do this? :)

**Comment by Romanitho** [comment]
Hi,

You can import ADMX and create a policy to manage the included apps list. This topic has already been discussed, and you can easily find an answer in the discussions and issues.
